radicle/src/storage/git/transport/local/url.rs

```diff
@@ -0,0 +1,106 @@
+//! Git local transport URLs.
+use std::str::FromStr;
```

Comment:
Was it considered to call the modules after their schemes?  'rad'/'heartwood' versus 'local'/'remote'?  The scheme names are unlikely to change, and its more transparent on the filesystem level about which scheme is held where.  Adding new schemes, even if unlikely, would be straightforward too.

Reply:
I like that idea :)

Reply:
yeah I considered that, but I chose a more descriptive approach, ie. naming them around their use, as especially for the remote transport, the scheme name `heartwood` is really just an internal detail (I was considering naming it `internal` at some point).